scripts/imgtool/image.py

```diff
@@ -275,6 +279,12 @@ def create(self, key, enckey, dependencies=None):
             # Add the size of the TLV info header
             protected_tlv_size += TLV_INFO_SIZE
 
+        if custom_protected_tlv:
+            if protected_tlv_size == 0:
+                protected_tlv_size = TLV_INFO_SIZE
```

Comment:
Please remove this, the `protected_tlv_size` is later increased by TLV_INFO_SIZE if necessary (protected_tlv_size != 0).